test/decl/class/actor/conformance.swift

```diff
@@ -2,6 +2,8 @@
 
 // REQUIRES: concurrency
 
+import Distributed
```

Comment:
See the comment about adding distributed to this file. For organization purposes, I think there is a better place.

Reply:
Yeah this is ok I think though you need to add the REQUIRES: distributed now; i commented above